srcpkgs/i3lock-next/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'i3lock-next'
+pkgname=i3lock-next
+version=0.1
+revision=1
+build_style=gnu-makefile
+hostmakedepends="git"
+makedepends="imlib2-devel fontconfig-devel libXrandr-devel"
+depends="i3lock-color libXrandr fontconfig bash"
+short_desc="Create a fancy image to use with i3lock"
+maintainer="Sinésio Neto <sinetoami@gmail.com>"
+license="MIT"
+homepage="https://github.com/owenthewizard/i3lock-next"
+
+do_fetch() {
+	git clone "https://github.com/owenthewizard/${pkgname}.git" "${wrksrc}"
```

Comment:
This is not versioned and will always download the latest git version.

Use a distfile with proper versioning, if no versioning exists pester upstream to do a release.

Reply:
@maxice8 excuse me, but I don't understand you when you said there about versioning (sorry for my bad english). Can you explain to me what I do with the `version` variable when creating git versions of packages? This [wiki](https://wiki.voidlinux.eu/Create_Git_Versions_of_Packages) didn't say and I didn't find it in the Manual.... sorry.

Reply:
Preferred is to get the upstream to tag a release. You just picked a version that doesn't match the upstream version

Reply:
Well, but this repo doesn't have a tag or release. Can I set with the date of the latest commit?

This is the repo
https://github.com/owenthewizard/i3lock-next

Reply:
set version to `0.0.YYYY.MM.DD`. Instead of running `git clone` pin your distfile to that commit. 

Something like:

~~~
_gitver=1c40be436dbd953ff11c41bb36ac34618ae525b5
distfile="https://github.com/owenthewizard/i3lock-next/archive/${_gitver}.tar.gz"
~~~

Preferred is still to get the upstream to tag a release.